subprojects/docs/src/docs/release/notes.md

```diff
@@ -282,6 +318,15 @@ For output properties annotated with [`@OutputFiles`](javadoc/org/gradle/api/tas
 In other words, if the property changes from `[file1, file2]` to `[file2, file1]` the task will not be up-to-date. 
 Prefer annotating individual properties with [`@OutputFile`](javadoc/org/gradle/api/tasks/OutputFile.html) and [`@OutputDirectory`](javadoc/org/gradle/api/tasks/OutputDirectory.html) if you can, or return a `Map` annotated with [`@OutputFiles`](javadoc/org/gradle/api/tasks/OutputFiles.html) or [`@OutputDirectories`](javadoc/org/gradle/api/tasks/OutputDirectories.html).
 
+### Avoid checking other repositories when dependency resolution in one repository fails
```

Comment:
:x: I believe we should explain how or if this works with repository blacklisting. A specific example may be helpful here.

Reply:
@eriwen The diff for this PR is messed up, and contains stuff that is not part of this PR. Will be fixed tomorrow. I think you want to talk to @blindpirate about #3047 instead.